Protocol/Roslyn/QActionSourceText.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Skyline.DataMiner.CICD.Models.Protocol
+{
+    using System;
+    using System.Text;
+
+    using Microsoft.CodeAnalysis.Text;
+
+    using Skyline.DataMiner.CICD.Models.Protocol.Read;
+
+    internal class QActionSourceText : SourceText
```

Comment:
Currently this is internal, but will be used in DIS for 'NavigateTo' in the validator & comparer. Unless you have thought of another way to achieve the same result in DIS?

Reply:
made public